checker/checker.ml

```diff
@@ -208,7 +208,7 @@ let usage () =
 open Type_errors
 
 let anomaly_string () = str "Anomaly: "
-let report () = (str "." ++ spc () ++ str "Please report" ++
+let report () = (str "." ++ spc () ++ str "Please report " ++
```

Comment:
Well, I would use indeed `str (". Please report at " ^ Coq_config.wwwbugtracker.")` `strbrk` adds some break hint, but IMHO it should become deprecated in favor of a paragraph construct. If you want to add it, the proper place for it at the top-level, thus:
```
strbrk (str (". Please report at " ^ Coq_config.wwwbugtracker."))
```

Reply:
No more `spc ()`? Whatever that does...